assets/src/components/Digitizing.js

```diff
@@ -414,7 +585,11 @@ export default class Digitizing extends HTMLElement {
         if (this.toolSelected === DigitizingAvailableTools[0]) {
             bootstrap.Dropdown.getOrCreateInstance(event.currentTarget).toggle();
         } else {
-            mainLizmap.digitizing.toolSelected = (mainLizmap.digitizing.toolSelected !== DigitizingAvailableTools[0]) ? DigitizingAvailableTools[0] : this.toolSelected;
+            if (mainLizmap.digitizing.toolSelected !== DigitizingAvailableTools[0]) {
+                mainLizmap.digitizing.toolSelected = DigitizingAvailableTools[0];
+            } else {
+                mainLizmap.digitizing.toolSelected = this.toolSelected;
+            }
```

Comment:
const firstAvailableTools =  DigitizingAvailableTools[0];
        if (this.toolSelected === firstAvailableTools) {
            bootstrap.Dropdown.getOrCreateInstance(event.currentTarget).toggle();
        } else if (mainLizmap.digitizing.toolSelected !== firstAvailableTools) {
            mainLizmap.digitizing.toolSelected = firstAvailableTools;
        } else {
            mainLizmap.digitizing.toolSelected = this.toolSelected;
       }